images/build/Dockerfiles/Dockerfile

```diff
@@ -77,6 +77,7 @@ RUN set -ex \
     && ln -s 10.14.2 10.14 \
     && ln -s 10.21.0 10.21 \
     && ln -s 12.18.2 12.18 \
+    && npm install -g lerna \
```

Comment:
Which image tag is Terrapin using? I ask because depending on the tag they are using, this might not be present in some tags which results in breaking the buld.

Reply:
Terrapin is using lts-versions image, so we'll test lerna in oryx/build:lts-versions image instead of full image now. And for the github-actions image, we don't have node/npm installed in image itself, I'm thinking add at least one node/npm version to it for testing, but that will increase image size/layers.

Reply:
> I'm thinking add at least one node/npm version to it for testing, but that will increase image size/layers.

We were discussing about updating the Node path to point to the root folder of the repo's node_modules folder. Have you checked to see if this helps in `lerna` being available to sub project's builds?

Reply:
Users could specify version of lerna and we could install it accordingly but still need to be global, locally didn't work. And here is the path info /opt/nodejs/12.18.3/bin/lerna -> /opt/nodejs/12.18.3/lib/node_modules/lerna/cli.js. I think it's already point to node_modules.
```
root@d0101a29df7f:/# npm install -g lerna@3.22.1
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated mkdirp-promise@5.0.1: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
**/opt/nodejs/12.18.3/bin/lerna -> /opt/nodejs/12.18.3/lib/node_modules/lerna/cli.js**
+ lerna@3.22.1
added 749 packages from 387 contributors in 20.946s
```